flink-runtime/src/main/resources/log4j.properties

```diff
@@ -18,7 +18,7 @@
 
 
 # Convenience file for local debugging of the JobManager/TaskManager.
-log4j.rootLogger=OFF, console
+log4j.rootLogger=INFO, console
```

Comment:
I'll very soon fix that issue so that we don't have to change the logging level every time we want to debug something.